defender/utils.py

```diff
@@ -202,8 +202,8 @@ def unblock_username(username, pipe=None):
         pipe = REDIS_SERVER.pipeline()
         do_commit = True
     if username:
-        pipe.delete(get_username_attempt_cache_key(username))
-        pipe.delete(get_username_blocked_cache_key(username))
+        pipe.delete(get_username_attempt_cache_key(username.lower()))
+        pipe.delete(get_username_blocked_cache_key(username.lower()))
```

Comment:
hey sorry guy's I guess we all dropped the ball on this one.  My apologies for not seeing this sooner as well!